_docs/index.md

```diff
@@ -18,4 +18,4 @@ The Lounge started as a fork of [Shout](https://github.com/erming/shout), author
 
 Enjoy The Lounge, and make sure to join us on our **official Freenode channel `#thelounge`**!
 
-*[IRC]: Internet Relay Chat
+{% include abbreviations.md %}
```

Comment:
I couldn't find a better way to make this a thing in all files for a given layout or whatnot, so until something arises, this is better than nothing I think.